tracing/tracing.go

```diff
@@ -67,10 +67,11 @@ func OverrideTraceResourceName(sourceCtx context.Context, newResourceName string
 //	    "GetMyModel.SQL",
 //	    "GetMyModel",
 //	)
-func ExecuteWithTrace[T any](ctx context.Context, callable func() (T, error), source, op string) (T, error) {
+func ExecuteWithTrace[T any](ctx context.Context, callable func(context.Context) (T, error), source, op string) (T, error) {
```

Comment:
Just would like to sync here so that we are on the same page.

So if someone calls `ExecuteWithTrace` what lambda "callable" they will pass and what they would do with context? 

What use case of that for them? How from the library we can explain it? 
Because I think most people will do "_ context.Context" in lambda
- We need expand documentation then